subprojects/core/src/main/groovy/org/gradle/api/file/CopySpec.java

```diff
@@ -324,4 +324,17 @@
      * {@inheritDoc}
      */
     CopySpec eachFile(Closure closure);
+
+    /**
+     * Gets the character encoding used when filtering files, to read and rewrite them.
+     * By default, the platform default encoding is used.
+     */
+    String getFilteringCharset();
+
+    /**
+     * Specifies the character encoding used when filtering files, to read and rewrite them.
+     *
+     * @param charset the name of the charset to use when filtering files
+     */
+    void setFilteringCharset(String charset);
```

Comment:
Let's use *charset* everywhere instead of mixing *charset* and *encoding*.
```
Gets the charset used to read and write files when filtering.
By default, the JVM default charset is used.
```
and
```
Specifies the charset used to read and write files when filtering.
```